Assets/SEE/Controls/KeyBindings.cs

```diff
@@ -311,8 +312,29 @@ public static void SetBindingForKey(KeyActionDescriptor descriptor, KeyCode keyC
             }
             else
             {
+                KeyCode oldKey = descriptor.KeyCode;
                 descriptor.KeyCode = keyCode;
+                string path = Application.dataPath + "/StreamingAssets/KeyBindings.json";
+                string jsonContent = File.ReadAllText(path);
+                List<KeyCode> jsonObject = JsonConvert.DeserializeObject<List<KeyCode>>(jsonContent);
+                int index = jsonObject.IndexOf(oldKey);
+                jsonObject[index] = keyCode;
+                File.WriteAllText(path, JsonConvert.SerializeObject(jsonObject));
+            }
+        }
+
+        /// <summary>
+        /// Returns a list of all keyCodes.
+        /// </summary>
+        /// <returns>all keyCodes</returns>
+        internal static List<KeyCode> GetKeyCodes()
```

Comment:
Whenever possible, interfaces should be used. Better use `IList` than `List`. If the order does not matter, you could even use `ICollection`.